projects/cashmere/src/lib/navbar/navbar.component.html

```diff
@@ -1,22 +1,22 @@
-<nav class="hc-navbar" [ngClass]="{'fixed-top': fixedTop}">
+<nav #navbar class="hc-navbar" [ngClass]="{'fixed-top': fixedTop}">
     <div class="navbar-brand">
         <a [routerLink]="homeUri" class="brand">
             <img src="{{brandIcon}}">
         </a>
     </div>
     <div class="navbar-app" [ngClass]="{'logo-condense': _logoCondense}">
         <a [routerLink]="homeUri" class="app" *ngIf="appIcon">
-            <img src="{{appIcon}}">
+            <img src="{{appIcon}}" (load)="appIconLoaded()">
         </a>
     </div>
-    <div class="hc-navbar-link-container">
+    <div #navlinks class="hc-navbar-link-container">
         <ng-content select="hc-navbar-link"></ng-content>
         <div class="hc-navbar-more-links" title="More" *ngIf="_collapse" [hcPopover]="navbarMore" popperPlacement="bottom">
             More<hc-icon fontSet="fa" fontIcon="fa-chevron-down" hcIconSm></hc-icon>
         </div>
     </div>
 
-    <div class="hc-navbar-right-container">
+    <div #rightcontainer class="hc-navbar-right-container">
```

Comment:
Should we prefix any of these ids? I could see a possible collision on this one

Reply:
This is an Angular variable name, and is scoped into the component.  There won't be any collisions (that would result in a compile error).

Reply:
That's a good call @jonest - maybe #hc-rightcontainer

Reply:
Doh! Good call @joeskeen :) I completely mixed the two up :) I saw the `#` and thought "jQuery ID tag!" 😆

Reply:
Again, not necessary - it's just a template variable.  I don't think you can even put hyphens in those.